coremltools/test/neural_network/test_compiled_model.py

```diff
@@ -88,3 +91,27 @@ def test_non_default_compute_units(self):
         for cur_compute_unit in non_default_compute_units:
             compiled_model_path = compile_model(self.spec)
             self._test_compile_model_path(compiled_model_path, compute_units=cur_compute_unit)
+
+
+    def test_destination_path_parameter(self):
+        # Check correct usage
+        with TemporaryDirectory() as temp_dir:
+            dst_path = temp_dir + "/foo.mlmodelc"
+            compiled_model_path = compile_model(self.spec, dst_path)
+            self._test_compile_model_path(compiled_model_path)
+
+        # Check bad input
+        with TemporaryDirectory() as temp_dir:
+            dst_path = temp_dir + "/foo.badFileExtension"
+            with pytest.raises(Exception) as e:
```

Comment:
nit: Change the general `Exception` to the corresponding specific exception (ValueError) as suggested in another comment in `utils.py`.

Reply:
Done.